applications/tari_app_grpc/proto/types.proto

```diff
@@ -76,4 +129,28 @@ message ConsensusConstants {
     uint64 block_weight_outputs = 15;
     /// Block weight for kernels
     uint64 block_weight_kernels = 16;
+    /// This is to keep track of the value inside of the genesis block
+    uint64 faucet_value = 17;
+    /// Maximum byte size of TariScript
+    uint64 max_script_byte_size = 18;
+    /// How long does it take to timeout validator node registration
+    uint64 validator_node_timeout = 19;
+    /// The height at which these constants become effective
+    uint64 effective_from_height = 20;
+    /// Current version of the blockchain
+    repeated uint64 valid_blockchain_version_range = 21;
```

Comment:
I think a Range type is better here
```suggestion
    Range valid_blockchain_version_range = 21;
    // ...
    
    message Range { 
      uint64 min = 1;
      uint64 max = 2;
    }
```

Reply:
Definitely ! I was unaware of the `Range` type, but it is much better, thanks for pointing it out !

Reply:
You'd have to create the type in proto :)